pallets/doas/Cargo.toml

```diff
@@ -36,3 +36,4 @@ std = [
 	"frame-support/std",
 	"frame-system/std",
 ]
+try-runtime = ["frame-support/try-runtime"]
```

Comment:
I guess this is the one you've mentioned in the stand up?

Reply:
yeah and it doesn't work. I'm going to merge this and fix it in another PR